bedrock/contentful/templates/includes/contentful/all.html

```diff
@@ -134,10 +134,12 @@ <h2 class="mzp-c-section-heading">{{ entry.heading }}</h2>
         mobile_class=entry.mobile_class,
         image_url=entry.image,
         include_highres_image=False,
-        loading=loading
+        loading=loading,
```

Comment:
**issue (blocking):** split default is to stack image before text on mobile but hero shows image after text on mobile

**suggestion:** content model update to allow setting the "media_after" param on split components OR hardcode bedrock's Contentful split call to set "media_after" param to true by default for all Contentful split components
<img width="800" alt="Screen Shot 2022-06-07 at 11 59 26 AM" src="https://user-images.githubusercontent.com/19650432/172364635-1e2f3d1e-5a0f-42c9-9a68-a13a7dee5448.png">